### PR TITLE
feat(cli/init): write file preserve order and print result in console

### PR DESCRIPTION
**Pull request by alexzhang1030**

### CODE REVIEW

File: crates/package_json/src/lib.rs [hunk 1]
```diff
@@ -99,7 +104,8 @@ impl PackageJson {
         if path.exists() {
             return Err(PackageJsonError::AlreadyExist);
         }
-        PackageJson::write_to_file(path)?;
+        let package_json = PackageJson::write_to_file(path)?;
+        println!("Wrote to {}\n\n{}", path.display(), serde_json::to_string_pretty(&package_json).unwrap_or(String::new()));
```
Comment (anonrig) [inline crates/package_json/src/lib.rs:108]: Nice!

**Review by anonrig — APPROVED** [review]

**Comment by anonrig** [comment]
I'm ok with merging this, but as a follow-up can you add a `insta` snapshot test for the output generated for package.json?

**Comment by anonrig** [comment]
It seems the formatter is failing. Can you run `just fmt`? Ref: https://github.com/casey/just

**Comment by alexzhang1030** [comment]
@anonrig Hey, I'm working on adding a snapshot for the initial package json. Would you mind if I open a new PR to do this?

**Comment by anonrig** [comment]
> @anonrig Hey, I'm working on adding a snapshot for the initial package json. Would you mind if I open a new PR to do this?

New PR is very much preferred :-)

**Pull request merged**